src/lib/source-file-converter.ts

```diff
@@ -912,6 +913,45 @@ export class SourceFileConverter {
     );
   }
 
+  /**
+   * Returns true if `node` is an identifier that is originally declared in
+   * a file called `fileName`.
+   *
+   * @param node The node to check. It can be any node type but this will return
+   *     false for non-Itentifiers.
+   * @param fileName Either a single filename, or an array of filenames. An
+   *     array is allowed because symbols may be declared in any of several file
+   *     types, for example: `.js`, `.ts`, or `.d.ts`.
+   */
+  isDeclaredIn(node: ts.Node, fileName: string | string[]) {
```

Comment:
Is this check good enough? This just checks that the node is from this module, but we want to know that it's a particular export of the module. Instead, could we get `html` from `lit-html` and check that the nodes are equivalent?

Reply:
Talked IRL. In 19a0fde, I resolve 'lit-html' and 'lit-element' once at the beginning of conversion and find symbols like `html` and `property`. During conversion, we can compare strict equality against these symbols. 👍